layouts/releases/list.html

```diff
@@ -6,16 +6,32 @@
                 <header>
                     <h1>Releases</h1>
                 </header>
+                {{ $releases := where .Site.Pages "Section" "releases" }}
                 {{ range .Pages }}
+                {{ $newerReleases := where $releases "Date" "gt" .Date }}
                 <article class="mt-4">
                     <span><a href="{{.Permalink}}"><h2>{{.Title}}</h2></a></span>
+                    <small>
+                        <i class="fa fa-fw fa-calendar"></i> Released on <b><time datetime="{{ .Params.date }}">{{ .Params.date.Format "2006-01-02 03:04:05 PM" }}</time></b>
```

Comment:
can we get just the date without the time? I don't think the exact time is relevant to anyone.

Reply:
Sure. That's easy. But to be honest I don't care if there is a time or not.
What do the others say?
React with :+1:  for "with time" and :-1:  for "without time".

Reply:
@ccoenen @davidmehren @DerMolly @InnayTool @ErikMichelson

Reply:
I'm indifferent. Usually, time should not matter. I can see how there might be two releases on a given day, but then you still have the version number. My initial question was more of a "oh dang, I didn't update, yet, how long did I not check that". And sometimes I use release dates in projects, to see if they were recently active. All of which "date only" solves.

Reply:
Oh, but _if_ there is time, it should absolutely by 24h hour clock ;-)

Reply:
I removed the time information